File: C/Daily Flash/Week2/Day4_Jan23/code/prog5.c
/*Program 5: Write a C program to calculate the factorial of a given number.
Input: 5
Output:
The Factorial of 5 is: 120
*/
#include <stdio.h>

void main() {
	int num, fact= 1;
	printf("Enter num for factorial: ");
	scanf("%d",&num);

	for(int i=1; i<=num; i++)
		fact *=i;
	printf("Factorial of %d : %d\n",num,fact);

}

File: JAVA/Daily Flash Codes/Week2/Day4_Jan23/code/prog5.java


import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));		//obj of Bufferedreader to take input
 		int num, fact=1;
		System.out.print("Enter num for factorial calculation: ");
		num = Integer.parseInt(in.readLine());	
		for(int lc = 1; lc<=num; lc++) 
			fact *=lc;
		System.out.println("Factorial of "+ num + " : " + fact );
	

	}


}
